src/mixer/samplerbank.cpp

```diff
@@ -10,8 +11,17 @@
 #include "track/track.h"
 #include "util/assert.h"
 
-SamplerBank::SamplerBank(PlayerManager* pPlayerManager)
+namespace {
+
+const ConfigKey kConfigkeyLastImportExportDirectory(
+        "[Samplers]", "lastImportExportDirectory");
```

Comment:
```suggestion
        "[Samplers]", "last
_import_export_directory");
```
New COs and config keys should use snake case.

Reply:
snake_case, now I got it, sry

Ready.